BUILD.gn

```diff
@@ -642,12 +640,22 @@ source_set("electron_lib") {
   ]
   if (enable_electron_extensions) {
     sources += filenames.lib_sources_extensions
+    deps += [
+      "shell/browser/extensions/api:api_registration",
+      "shell/common/extensions/api",
+      "shell/common/extensions/api:extensions_features",
+      "//chrome/browser/resources:component_extension_resources",
```

Comment:
It is needed, otherwise:
```
Undefined symbols for architecture x86_64:
  "_kComponentExtensionResources", referenced from:
      extensions::ElectronComponentExtensionResourceManager::ElectronComponentExtensionResourceManager() in electron_component_extension_resource_manager.o
      extensions::ElectronComponentExtensionResourceManager::ElectronComponentExtensionResourceManager() in electron_component_extension_resource_manager.o
  "_kComponentExtensionResourcesSize", referenced from:
      extensions::ElectronComponentExtensionResourceManager::ElectronComponentExtensionResourceManager() in electron_component_extension_resource_manager.o
      extensions::ElectronComponentExtensionResourceManager::ElectronComponentExtensionResourceManager() in electron_component_extension_resource_manager.o
```